packages/graphql-transformer-core/tsconfig.json

```diff
@@ -4,7 +4,6 @@
         "module": "commonjs",
         "sourceMap": true,
         "outDir": "lib",
-        "sourceRoot": "src",
```

Comment:
Why did we remove this?

Reply:
It is causing a problem when debugging in VSCode, no need for that. With that setting you cannot debug the TS files, breakpoints were not hit, just the compiled JS files.